pyproject.toml

```diff
@@ -2,6 +2,14 @@
 requires = ["setuptools", "wheel"]  # PEP 508 specifications.
 
 
+[tool.ruff]
```

Comment:
You may want to add https://github.com/aws/aws-lambda-builders/blob/develop/pyproject.toml#L4 here too. Pylint is off by default.

Reply:
Good to know!

Reply:
😎  sam-t is away ahead of cli regarding how many rules it comply: https://github.com/aws/serverless-application-model/blob/develop/ruff.toml#L4